### PR TITLE
Add design section to the handbook

### PR DESCRIPTION
**Pull request by christinaforney**

### CODE REVIEW

**Comment by imtommyroberts** [comment]
Thoughts on having the product design page be a subset of the design page, and also having a page for graphic or brand designers?

**Comment by christinaforney** [comment]
> Thoughts on having the product design page be a subset of the design page, and also having a page for graphic or brand designers?

If you have enough content to fill out those pages now go for it! Otherwise, we don't need to over optimize early when there won't be anyone using those sections for awhile.

File: handbook/design/index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Product Design
+
+The role of the product design team is to make validated decisions that enhance users experience of Sourcegraph.
```
Comment (christinaforney) [inline handbook/design/index.md]:
```suggestion
The role of the product design team is to make validated decisions that enhance the user experience of Sourcegraph.
```

**Comment by imtommyroberts** [comment]
Awesome! What do I need to do to make it available in the handbook?

**Comment by christinaforney** [comment]
Click this button, and it'll automatically get merged into master, and then get deployed! You're safe to do this because you have two things:
1. Approval on the PR
2. A green check mark next to "All checks have passed" and "This branch has no conflicts"

image

**Pull request merged**